Gemfile

```diff
@@ -0,0 +1,55 @@
+source 'https://rubygems.org'
+
+
+gem 'rails', '4.2.5'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.1.0'
+gem 'jquery-rails'
+gem 'turbolinks'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.0'
+# bundle exec rake doc:rails generates the API under doc/api.
+gem 'sdoc', '~> 0.4.0', group: :doc
+
+# bootstrap for styling
+gem 'bootstrap-sass'
+
+# Use ActiveModel has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+gem 'omniauth'
+gem 'omniauth-oauth2', '~> 1.3.1'
+gem "omniauth-vimeo"
+gem 'omniauth-twitter'
+
+gem 'httparty'
+gem 'twitter'
+gem 'will_paginate', '~> 3.0.5'
+
+group :development, :test do
+  gem 'byebug'
+  gem 'dotenv-rails'
+  gem 'sqlite3'
+  gem 'pry'
+  gem 'simplecov'
+end
+
+group :production do
+  gem 'pg'
+  gem 'rails_12factor'
+end
+
+group :test do
+  gem 'shoulda-matchers', '~> 3.0'
+  gem "factory_girl_rails", "~> 4.0"
+end
+
+group :development do
+  gem 'httplog'
+  gem 'web-console', '~> 2.0'
+  gem 'spring'
+  gem 'better_errors'
+  gem 'binding_of_caller'
+  gem 'rspec-rails'
+end
```

Comment:
Nicely organized and easy to read but comments about what the gems do can be helpful to someone who is unfamiliar with a particular gem.  I checked out 'shoulda-matchers' and it seems very cool.